src/sections/Landscape/ServiceMeshTimeline.js

```diff
@@ -46,7 +46,7 @@ const ServiceMeshTimeline = ( { theme } ) => {
             <img src={theme === "dark" ? mesh?.darkIcon || mesh.icon : mesh.icon } alt={mesh.name} />
           </div>
           :   <div className={`meshtitle-img-${mesh.timeline_order % 2}`}>
-            <img src={ServiceMeshIcon} alt={mesh.name} />
+            <img src={ServiceMeshIcon} alt={mesh.name} class={mesh.name} />
```

Comment:
This would unnecessarily add classnames for all the images, please update it to add only for "Vulcand".